frontends/web/src/components/sidebar/sidebar.tsx

```diff
@@ -203,70 +204,70 @@ class Sidebar extends Component<Props> {
-          <div className="sidebarHeaderContainer end"></div>
+          <div className={[style.sidebarHeaderContainer, style.end].join(' ')}></div>
           { accounts.length ? (
-            <div key="buy" className="sidebarItem">
+            <div key="buy" className={style.sidebarItem}>
               <NavLink
-                className={({ isActive }) => isActive ? 'sidebar-active' : ''}
+                className={({ isActive }) => isActive ? style['sidebar-active'] : ''}
                 to="/buy/info"
               >
-                <div className="single">
-                  <img draggable={false} className="sidebar_settings" src={coins} alt={t('sidebar.exchanges')} />
+                <div className={style.single}>
+                  <img draggable={false} className={style['sidebar_settings']} src={coins} alt={t('sidebar.exchanges')} />
                 </div>
-                <span className="sidebar_label">
+                <span className={style['sidebar_label']}>
                   {t('sidebar.buy')}
                 </span>
               </NavLink>
             </div>
           ) : null }
           { deviceIDs.map(deviceID => (
-            <div key={deviceID} className="sidebarItem">
+            <div key={deviceID} className={style.sidebarItem}>
               <NavLink
                 to={`/device/${deviceID}`}
-                className={({ isActive }) => isActive ? 'sidebar-active' : ''}
+                className={({ isActive }) => isActive ? style['sidebar-active'] : ''}
                 title={t('sidebar.device')}
                 onClick={this.handleSidebarItemClick}>
-                <div className="single">
-                  <img draggable={false} className="sidebar_settings" src={deviceSettings} alt={t('sidebar.device')} />
+                <div className={style.single}>
+                  <img draggable={false} className={style['sidebar_settings']} src={deviceSettings} alt={t('sidebar.device')} />
                 </div>
-                <span className="sidebar_label">{t('sidebar.device')}</span>
+                <span className={style['sidebar_label']}>{t('sidebar.device')}</span>
               </NavLink>
             </div>
           )) }
-          <div key="settings" className="sidebarItem">
+          <div key="settings" className={style.sidebarItem}>
             <NavLink
-              className={({ isActive }) => isActive ? 'sidebar-active' : ''}
+              className={({ isActive }) => isActive ? style['sidebar-active'] : ''}
               to={'/settings'}
               title={t('sidebar.settings')}
               onClick={this.handleSidebarItemClick}>
-              <div className="stacked">
-                <img draggable={false} className="sidebar_settings" src={settingsGrey} alt={t('sidebar.settings')} />
-                <img draggable={false} className="sidebar_settings" src={settings} alt={t('sidebar.settings')} />
+              <div className={'stacked'}>
```

Comment:
could change to `<div className="stacked">`, but ideally it is not a global see other comment.